.github/workflows/build-publish-develop.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Build Chainlink and push image with latest develop tag'
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-publish-chainlink-develop:
```

Comment:
Ditto, something similar here so its easier to see jobs ran at a glance

Reply:
Updated to `push-chainlink-develop`